pkgs/by-name/me/metadata/package.nix

```
@@ -0,0 +1,39 @@
+{ stdenv, lib, fetchFromGitHub, pkg-config, ffmpeg, rustPlatform, asciidoc, libavutil }:
+rustPlatform.buildRustPackage rec {
+  pname = "metadata";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "zmwangx";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-OFWdCV9Msy/mNaSubqoJi4tBiFqL7RuWWQluSnKe4fU=";
+  };
+
+  cargoHash = "sha256-F5jXS/W600nbQtu1FD4+DawrFsO+5lJjvAvTiFKT840=";
+
+  # Build-time
+  nativeBuildInputs = [ pkg-config asciidoc ];
+
+  postBuildHook = "a2x --doctype manpage --format manpage man/metadata.1.adoc";
+  postInstallHook = ''
+    mkdir -p $out/share/man
+    cp man/metadata.1 $out/share/man/
+  '';
+
+  # Run-time AND build-time
+  BuildInputs = [ ffmpeg libavutil ];
```

```suggestion
  buildInputs = [ ffmpeg libavutil ];
```
`buidInputs` is lowercase. Also unnessary explanation, maintainers should know what this does.